_styles/post-excerpt.scss

```diff
@@ -1,31 +1,69 @@
 ---
 ---
 
+$thumb-size: 200px;
+$wrap: 800px;
+
+.post-excerpt-container {
+  container-type: inline-size;
+}
+
 .post-excerpt {
   display: flex;
-  flex-wrap: wrap;
-  gap: 20px;
   margin: 20px 0;
-  padding: 20px 30px;
   border-radius: var(--rounded);
   background: var(--background);
-  text-align: left;
+  overflow: hidden;
   box-shadow: var(--shadow);
 }
 
-.post-excerpt > * {
+.post-excerpt-image {
+  position: relative;
+  width: $thumb-size;
+  flex-shrink: 0;
+  // box-shadow: var(--shadow);
```

Comment:
Just in case, consider removing this if it's no longer needed.

Reply:
Some styles are kept so users can easily uncomment them out. Think of it like an "alternative" style that we don't necessarily want to bring out into the config file, but still want to allow for with minimal hassle.